zhook.py

```diff
@@ -369,6 +414,15 @@ def dumpJson(self):
     print("ERROR: no Ziti identity provided, set INPUT_ZITIID or INPUT_ZITIJWT")
     exit(1)
 
+  # Validate zitiId as JSON and print debug logs for top-level keys
+  try:
+    zitiIdJson = json.loads(zitiId)
```

Comment:
Is it an issue to emit keys here? Are they visible by external people and/or can they be used for accessing anything?